doc/courses/sensys/exercises/capsule/src/sensys.rs

```diff
@@ -9,8 +9,9 @@
 extern crate kernel;
 
 use kernel::hil::sensors::{AmbientLight, AmbientLightClient};
-use kernel::hil::time::{self, Alarm, Frequency};
+use kernel::hil::time::{self, Alarm};
```

Comment:
[The tutorial](https://github.com/tock/tock/blob/master/doc/courses/sensys/capsule.md#51-when-is-now-when-is-one-second-from-now) uses the frequency trait. So if you're going to remove it, please update the tutorial text to tell users to add it back in when needed.

Reply:
Better to add `#[allow(unused)]` around that import

Reply:
Cool. I didn't know you could do that for imports too.